src/Controller/CartController.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace Helret\HelloRetail\Controller;
+
+use Helret\HelloRetail\Service\HelloRetailRecommendationService;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Storefront\Controller\StorefrontController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+
+#[Route(defaults: ['_routeScope' => ['storefront']])]
+class CartController extends StorefrontController
+{
+    public function __construct(
+        private readonly HelloRetailRecommendationService $helloRetailService,
+        private readonly SystemConfigService $systemConfigService
+    ) {
+    }
+
+    #[Route(
+        path: '/checkout/cart/recommendations',
+        name: 'frontend.cart.recommendations',
```

Comment:
@wexoigi I'm prette sure the convention is to use the plugin name in the path and name to avoid collision with core or 3rd party code, which these very generic names have a risk of.

See current Hello retail naming convention here:

https://github.com/helloretail/shopware6-extension/blob/master/src/Core/Content/Export/SalesChannel/FileController.php#L31